modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/common/components/donut-chart/index.js

```diff
@@ -13,13 +13,20 @@
  */
 
 import ClayChart from '@clayui/charts';
+import ClayIcon from '@clayui/icon';
 import React, {useEffect, useRef} from 'react';
 
+import ClayIconProvider from '../../context/ClayIconProvider';
+
 import './index.css';
 
 const DonutChart = ({
 	chartData,
+	dateUntilGoal,
```

Comment:
Please, it needs to be a generic chart. Change ```dateUntilGoal```.

Reply:
I changed that by refactoring the code.